doc/release-notes/release-notes-current.md

```diff
@@ -0,0 +1,25 @@
+zend v4.0.99
+=========
+
```

Comment:
Given that in this release we introduced a couple of changes potentially having quite an impact on users, I think it could be useful to add some notes to make it more explicit what they could expect from the next release, for instance something like this:

```suggestion

Important notes:
- PR [#541](https://github.com/HorizenOfficial/zen/pull/541) fixes an issue that made Explorers, relying on `zend` RPC commands, not to show old transactions including P2PK scripts. The fix will have an effect only on blocks processed after the update, so if you are running an explorer you need to run a (fast) reindex to fix how the old blocks affected are shown. However, reindexing is not mandatory.
- PR [#526](https://github.com/HorizenOfficial/zen/pull/526) implements [ZenIP-42204](https://github.com/HorizenOfficial/ZenIPs/blob/master/zenip-42204.md), which introduces a hard fork disabling the possibility to move transparent funds to shield addresses. After the hard fork, only shielded-to-shielded and shielded-to-transparent transactions will be allowed. For this reason, some RPC commands related to shielded addresses have been deprecated and may partially or completely stop working after the hard fork.

```

Reply:
Ok aligned based on your suggestion, with minor revisions and changes,